fairing/frameworks/lightgbm.py

```diff
@@ -86,12 +115,12 @@ def _generate_entrypoint(copy_files_before, copy_files_after, config_file,
         for rank, files in copy_patitioned_files.items():
             buf.append("\t{})".format(rank))
             buf.extend(
-                ["\t\t"+cmd for cmd in _get_commands_for_file_ransfer(files)])
+                ["\t\t" + cmd for cmd in _get_commands_for_file_transfer(files)])
```

Comment:
Why change here, the PR is only for update docString, for others, pls create another PR for this.

Reply:
will revert